test/endToEnd/scenarioTest.js

```diff
@@ -106,7 +106,41 @@ function switchInSmaa(browser) {
     .click('option[label="Default"]')
     .assert.containsText('#scenario-selector', 'Default');
 }
+function deleteScenario(browser) {
+  browser
+    .waitForElementVisible('#delete-scenario-disabled')
+    .assert.containsText('#scenario-selector', 'Default')
+    .click('#copy-scenario-button')
+    .waitForElementVisible('#create-new-scenario-button:disabled')
+    .setValue('#new-scenario-title', scenarioTitle)
+    .waitForElementVisible('#create-new-scenario-button:enabled')
+    .click('#create-new-scenario-button')
+    .pause(50) //pause needed to not get 'stale element' error
+    .waitForElementVisible('#delete-scenario-button')
+    .assert.containsText('#scenario-selector', scenarioTitle)
+    .click('#delete-scenario-button')
+    .waitForElementVisible('#delete-scenario-header')
+    .click('#delete-scenario-confirm-button')
+    .waitForElementVisible('#delete-scenario-disabled')
+    .assert.containsText('#scenario-selector', 'Default');
+}
 
```

Comment:
a test that the delete button is disabled when there's precisely 1 scenario would also be good I think; same for subproblem (forgot that yesterday)

Reply:
That is being checked inside the tests (here line 124), I don't feel like a separate test is necessary.